.travis.yml

```diff
@@ -124,12 +126,13 @@ matrix:
     # TODO: remove next before_install, install and script clause when the wheels become available
     before_install:
       - pyenv global $(pyenv whence 2to3)  # activate all python versions
-      - PY_CMD=python3
+      - PY_CMD=python3.8
```

Comment:
In case you see this before my reply on pybind11, try adding this package, which allows `python3.8 -m pip` to work:
```
          - python3.8-distutils
```

Reply:
Yup, just added it in - thanks!